### ISSUE TITLE
Support for InnoSetup 5.6.0

### ISSUE THREAD

**Issue by DoCode** [dscharrer/innoextract#62]

**Comment by crazy-max** [comment]
```
$ innoextract.exe -d C:\innosetup innosetup-5.6.0.exe
Stream error while parsing setup headers!
 ├─ detected setup version: 5.5.7
 └─ error reason: basic_ios::clear
If you are sure the setup file is not corrupted, consider 
filing a bug report at http://innoextract.constexpr.org/issues
```

**Comment by dscharrer** [comment]
This should be fixed with innoextract 1.7 (to be released soon), a [release candidate](http://constexpr.org/innoextract/files/snapshots/innoextract-1.7-rc1/) is available for testing.

**Comment by crazy-max** [comment]
Wow thanks @dscharrer ! Very efficient :D
